### ISSUE TITLE
[no-var-requires] Invalid version number identifier in context link

### ISSUE THREAD

**Issue by oliversalzburg** [typescript-eslint/typescript-eslint#3480]
In a TypeScript project, I have this line in a file:
```ts
const path = require("path");
```
`require("path")` will have red squiggly lines under it. If I hover over the element, I get a popover describing the issue as:
> Require statement not part of import statement.eslint@typescript-eslint/no-var-requires

The last part of which is a link that, when clicked, leads to an invalid URL:
```
https://github.com/typescript-eslint/typescript-eslint/blob/v%5Bobject%20Object%5D/packages/eslint-plugin/docs/rules/no-var-requires.md
```

![image](https://user-images.githubusercontent.com/1658949/120345805-192ea800-c2fb-11eb-95bd-f0cbdefb9f91.png)

Previously reported as https://github.com/microsoft/vscode-eslint/issues/1274

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [ ] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

https://github.com/oliversalzburg/vscode-eslint-version-repro

**Expected Result**

The link is valid.

**Actual Result**

It is not.

**Additional Info**

**Versions**

See the version information in the reproduction.

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `X.Y.Z` |
| `@typescript-eslint/parser`        | `X.Y.Z` |
| `TypeScript`                       | `X.Y.Z` |
| `ESLint`                           | `X.Y.Z` |
| `node`                             | `X.Y.Z` |

**Comment by KuryKat** [comment]
I was going to make a new issue to report the same error and I saw your issue, I confirm this bug, and it's not just for this "no-var-requires" rule, it's for all rules.

![image](https://user-images.githubusercontent.com/57152799/120660207-7a6e8c80-c45d-11eb-84e2-4da61e4c6aee.png)

![image](https://user-images.githubusercontent.com/57152799/120660262-88241200-c45d-11eb-9d24-9bdb3bfb040c.png)

I've changed the `"v[object Object]"` to `"master"` when I need to read the documentation, but they should fix it soon, considering that not everyone knows, or has the patience, to switch to "master" in the link (and because it's more feasible that the system works as it should, by pointing to the valid URL)

**Comment by bradzacher** [comment]
Duplicate of https://github.com/typescript-eslint/typescript-eslint/issues/3473

Please use the search next time

**Comment by oliversalzburg** [comment]
How nice of you to assume I didn't use the search 😉
